CONTRIBUTING.md

```diff
@@ -1,13 +1,63 @@
 # Contributing guidelines
 
-All Terraform Baseline repositories are maintained by the Terraform Baseline team.
+This document gives an overview of how to contribute to Terraform Baseline repositories.
 
-## Requesting changes
+## 💡 Requesting changes
 
 Open an issue in [this repository](https://github.com/equinor/terraform-baseline/issues/new/choose).
 
-## Making changes
+## 📝 Making changes
 
-1. Fork the relevant module repository.
+1. Create a new branch.
 1. Commit your changes.
 1. Create a pull request.
+
+## 🤝 Roles and responsibilities
+
+This section describes the various roles and responsibilities in the Terraform Baseline project.
+
+### 🦸‍♀️ External contributors
+
+An external contributor should:
+
+- Create forks of Terraform Baseline repositories.
+- Create issues and pull requests in Terraform Baseline repositories.
+
+### 👨‍🎓 Contributors
+
+A contributor must:
+
+- Be a member of the [@equinor/terraform-baseline](https://github.com/orgs/equinor/teams/terraform-baseline) team.
+- Actively contribute to one or more Terraform Baseline repositories.
+
+A contributor should:
+
+- Have basic understanding of Terraform.
+- Have basic understanding of Terraform Baseline best practices.
+- Create module repositories.
+
+### 👷‍♀️ Maintainers
+
+A maintainer must:
+
+- Be a member of the [@equinor/terraform-baseline-maintainers](https://github.com/orgs/equinor/teams/terraform-baseline-maintainers) team.
+- Review and approve pull requests in all Terraform Baseline repositories.
+- Discuss and update best practices when needed.
+
+A maintainer should:
+
+- Have complete understanding of Terraform.
+- Have complete understanding of Terraform Baseline best practices.
+- Communicate and share with other maintainers.
+
+### 👮‍♂️ Administrators
+
+An administrator must:
+
+- Be a member of the [@equinor/terraform-baseline-admins](https://github.com/orgs/equinor/teams/terraform-baseline-admins) team.
+
+An administrator should:
+
+- Configure module repositories.
+- Public modules to Terraform Registry.
+- Recruit people to the above roles 🤗
```

Comment:
Should we include that either Adminitrators or Administrators+Maintainers should also have a role for when to archive modules?

Reply:
According to [this overview](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-repository-roles/repository-roles-for-an-organization#permissions-for-each-role), only admins have the required permissions to archive repositories, so I'll add this to the responsibilities of `Administrators`.